app/controllers/users_controller.rb

```diff
@@ -0,0 +1,12 @@
+class UsersController < ApplicationController
+  before_action :current_user
+
+  def subscriptions
+    if @current_user.nil?
+      redirect_to login_path
+    else
+      @twitter_subscriptions = @current_user.twitter_users
+      @vimeo_subscriptions = @current_user.vimeo_users
+    end
```

Comment:
Another way you could write this is: 

```ruby
redirect_to login_path if @current_user.nil?`

@twitter_sub...
```

If the redirect is triggered because there isn't a user, the following two lines won't run. This saves you from doing an `if` block.